academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,116 @@
+True | 0 | Cultural influences in acquiescent response: A study of trainer evaluation biases | Cultural Influences in Acquiescent Response: A Study of Trainer Evaluation Biases
+True | 0 | Self-managers: Social contexts, personal traits, and organizational commitment | Self-managers: Social contexts, personal traits, and organizational commitment
+True | 0 | The meaningfulness of managerial work: case of Taiwanese employees | The meaningfulness of managerial work: case of Taiwanese employees
+True | 0 | When perceived welfare practices leads to organizational citizenship behavior | When perceived welfare practices leads to organizational citizenship behavior
+True | 0 | Negative Effects of Abusive Supervision: The Path Through Emotional Labor | Negative Effects of Abusive Supervision: The Path Through Emotional Labor
+True | -42 | The Mediating Mechanisms between Prosocial Motivation and Job Performance in Mission-driven Organizations | The mediating effect of calling and job involvement on prosocial motivation and performance
+False | -85 | Age and Gender Differences in Playfulness and Innovation Behavior among Professional Workers | THE INFLUENCE OF GENDER, AGE AND PROFESSION- RELATED DIFFERENCES ON THE VERBAL BEHAVIOR OF THE SUBJECTS OF ASSOCIATION
+True | -3 | Factors affecting training transfer | Factors Affecting Motivation to Transfer Training.
+True | 0 | A Review of Microsoft Academic Services for Science of Science Studies | A Review of Microsoft Academic Services for Science of Science Studies
+False | -60 | Survey on corporate motivations for culture and arts sponsorship | The Art of Co-Creative Media: An Australian Survey
+False | -309 | The Relationship among Transformational Leadership, Public Administrative Ethic and Organizational Citizenship Behavior: An empirical sturdy for tax bureau district office of Kaohsiung | Sharing The Fire: The igniting role of transformational leadership on the relationship between public managers’ and employees’ organizational commitment
+False | -81 | The relationship between Advisors’ Leadership Behaviors and Graduate Students’ Achievement Motives | The Relationship between Transformational Leadership Behaviors of Faculty Supervisors and Self-Efficacies of Graduate Assistants.
+False | -186 | The Effects of Budgetary Goal Difficulty, Task Uncertainty and Budgetary Emphasis on Performance: Moderated Regression Analysis | THE JOINT EFFECTS OF BUDGETARY SLACK AND TASK UNCERTAINTY ON SUBUNIT PERFORMANCE
+False | -95 | Promoting Participation in Performing Arts Activities through Internet Technology: A Case Study in Taiwan. | Enhancing students' learning experience and promoting inclusive practice via social media
+
+## Next steps
+
+Advance to the next section to learn how to design a MAKES schema to enable filter and search for the library data.
```

Comment:
>filter [](start = 76, length = 6)

filters?

Reply:
filter and search as verbs?

---
In reply to: [508614642](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/185#discussion_r508614642) [](ancestors = 508614642)